audio/templates/html/audio_edit.html

```diff
@@ -0,0 +1,82 @@
+        </div>
+        <div class="radio-group">
+            <label for="content_type">
+                <span class="bold-label">Select Audio Type:</span>
+            </label>
+            <div class="radio-container">
+                <label>
+                    <input type="radio" name="content_type" value="audio" onclick="showContent('audioTab')" id="audioRadio" checked>
+                    Audio file
+                </label>
+                <label>
+                    <input type="radio" name="content_type" value="podcast" onclick="showContent('podcastTab')" id="podcastRadio">
+                    Podcasts
+                </label>
+            </div>
+        </div>
+    
+        <div id="audioTab" class="tabcontent">
+            <form class="studio-xblock-form">
+                <div class="field">
+                    <label for="sources">Audio file URLs</label>
+                    <textarea id="sources" name="sources" {% if self.embed_url %}disabled{% endif %}>{{ self.sources }}</textarea>
+                    <p class="help">Place each URL on a new line.</p>
+                </div>
+                <div class="field">
+                    <label for="transcript_file">Audio transcript</label>
+                    <input type="file" id="transcript_file" name="transcript_file" />
+                    <p class="help">Only WebVTT format is supported.</p>
+                </div>
+                <div class="field">
+                    <label for="allow_audio_download">Allow audio download</label>
+                    <select id="allow_audio_download" name="allow_audio_download">
+                        <option value="true" {% if self.allow_audio_download %}selected{% endif %}>Yes</option>
+                        <option value="false" {% if not self.allow_audio_download %}selected{% endif %}>No</option>
+                    </select>
+                    <p class="help">You can allow students to download the source file of this audio.</p>
+                </div>
+                <div class="checkbox-container">
+                    <input type="checkbox" id="start_time_checkbox" name="start_time_checkbox" {% if self.start_time or self.end_time %}checked{% endif %}/>
+                    <label for="start_time_checkbox">Set start and end time</label>
+                </div>
+
+                <div class="time-fields" id="time_fields" {% if not self.start_time and not self.end_time %}style="display:none"{% endif %}>
+                    <div class="field">
+                        <label for="start_time">Start time</label>
+                        <input type="text" id="start_time" name="start_time" value="{{ self.start_time|default:"00:00" }}" />
+                    </div>
+                    <div class="field">
+                        <label for="end_time">End time</label>
+                        <input type="text" id="end_time" name="end_time" value="{{ self.end_time|default:"00:00" }}" />
```

Comment:
For an already configured xblock, these values are displayed  as seconds only and  not in the `mm:ss` format.